[ADD] sale_payment_mgmt: New module (#350)
OCA/account-payment: sale_payment_mgmt/readme/DESCRIPTION.rst

> This addon adds a Payments (only customer payments) menu in Sales to users are in some of this Sale groups:

Reviewer: ```suggestion
This addon adds a Payments (only customer payments) menu in Sales to users that belong to any of these Sales groups:
```

>     User: Own Documents Only

Reviewer: Leave empty blank line for proper RST syntax

>     User: Own Documents Only
>     User: All Documents
>     Administrator

Reviewer: As the names can be misleading, prefix it with Sales, and put properly lists:
```suggestion
* Sales / User: Own Documents Only
* Sales / User: All Documents
* Sales / Administrator
```

> .. image:: ../static/description/payment_menu.png

Reviewer: You can draw an arrow pointing to the menu entry.

Author: Image updated now

.. image:: ../static/description/payment_menu.png

> Show "Register payment" button in invoices (that allow more faster create and validate payment related) to users in some of this Sale groups:

Reviewer: ```suggestion
Show "Register payment" button in invoices (that allows a more faster creation and validation of related payments) to users in some of these above groups
```
(and remove again the same list)

>     User: Own Documents Only

Reviewer: Empty line

    User: Own Documents Only
    User: All Documents
    Administrator

> Hide 'Vendor' option in Partner type about create payments if user not in Invoicing group

Reviewer: No need to mention this, as it's already mentioned before that only customer payments.

> Hide 'Vendor' option in parther_type about payments manual create if user not in Invoicing group

Reviewer: Put this is usage section and use correct English, please.

> If user is in Sale group "User: Own Documents Only" only can view ourself payments (create is ourself)

Reviewer: ```suggestion
If users are in group "Sales / User: Own Documents Only", they can only view their payments (created by themselves).
```

> Note: This addon create some Record rules related to account_move and account_payment to allow taht work fine

Reviewer: account_move? They shouldn't...

Anyway, I think this is not needed IMO.
